### DISCUSSION TITLE
Error: Command "start" is not available outside of a Sanity project context…

### DISCUSSION THREAD

**Discussion by freshyill** [sanity-io/sanity#3576] [Sanity Studio Support]
I'm getting this error with a newly-created project (created using `npm create sanity@dev-preview`):

> Error: Command "start" is not available outside of a Sanity project context. Run the command again within a Sanity project directory, where "@sanity/core" is installed as a dependency.
>    at _.runCommand (/opt/homebrew/lib/node_modules/@sanity/cli/bin/sanity-cli.js:3608:1340)
>    at t.exports (/opt/homebrew/lib/node_modules/@sanity/cli/bin/sanity-cli.js:1980:2422)

After I create the project, I attempt to run it using `sanity start` from within the newly created studio directory.

All I'm doing is walking through the setup process, selecting an existing project to use with it. (previously created using the Eleventy Blog starter), and using the pre-made blog schema, nothing fancy.  I haven't even gotten past the [Quickstart](https://beta.sanity.io/docs/learn).

I don't see anything about an update to the CLI needed for the v3 studio or anything like that. Is there something obvious I'm doing wrong? 

Edit: I do see[ a discussion on Slack](https://sanity-io-land.slack.com/archives/C9Z7RC3V1/p1658109567869709) that mentions this error, but it seems to be something different.

**Comment by kmelve** [comment]
It seems to me that it's the Sanity Studio CLI v2 that runs here, and is confused by the lack of v2 dependencies. Could you try starting it using the npm scripts in the v3 folder, so `npm start`?

**Comment by pattyouwehand** [comment]
Hi! I have the exact same problem, has anyone found a solution for this yet?

`npm start` gives the same result.
